src/app.coffee

```diff
@@ -1,3 +1,5 @@
+Promise.config({ cancellation: true })
```

Comment:
You need to import bluebird before you try to enable cancellation on it

Reply:
I'll remove this since I'm going to implement your suggestion.